addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InsecureJsfViewStatePassiveScanRule.java

```diff
@@ -254,4 +241,23 @@ public int getCweId() {
     public int getWascId() {
         return 14; // WASC Id - Server Misconfiguration
     }
+
+    private AlertBuilder createAlert(String viewState) {
+        return newAlert()
+            .setRisk(getRisk())
+            .setConfidence(Alert.CONFIDENCE_LOW)
+            .setDescription(getDescription())
+            .setOtherInfo(Constant.messages.getString(MESSAGE_PREFIX + "extrainfo", viewState))
+            .setSolution(getSolution())
+            .setReference(getReference())
+            .setCweId(getCweId())
+            .setWascId(getWascId());
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+            createAlert("<input type='hidden' id='javax.faces.viewstate' value=1231'").build()
```

Comment:
The single quotes are mismatched/unbalanced.
Also shouldn't all the `'` be `\"`?

Reply:
The single quotes still aren't balanced